starknet-replay/src/error/database.rs

```diff
@@ -1,30 +1,44 @@
 //! This file contains the enum `Error` for all the errors returned by the
 //! module `pathfinder_db`.
 
+use pathfinder_storage::BlockId;
+use starknet_api::hash::StarkFelt;
 use thiserror::Error;
 
 #[derive(Debug, Error)]
 pub enum Error {
     /// `ConnectToDatabase` is used to encapsulate errors of type
     /// `anyhow::Error` which are originating from the
-    /// function `starknet_replay::pathfinder_db::connect_to_database`.
+    /// function `starknet_replay::runner::pathfinder_db::connect_to_database`.
     #[error(transparent)]
     ConnectToDatabase(anyhow::Error),
 
     /// `GetLatestBlockNumber` is used to encapsulate errors of type
     /// `anyhow::Error` which are originating from the
-    /// function `starknet_replay::pathfinder_db::get_latest_block_number`.
+    /// function `starknet_replay::runner::pathfinder_db::get_latest_block_number`.
```

Comment:
Mentions of actual functions and types should be done as rustdoc links so people can actually click through, not just monospaced text. There are more of these than I had spotted before, so please scan all your files and check for these!

Make sure you run rustdoc with private items on to check for and fix any errors.